<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpidrMpx3Eq</class>
 <widget class="QMainWindow" name="SpidrMpx3Eq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpidrMpx3Eq</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="_tabDACs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1101</width>
      <height>531</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Equalization&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Equalization</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>300</y>
        <width>241</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="_connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_startEq">
         <property name="text">
          <string>Start Eq</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="_statusLabel">
         <property name="text">
          <string>Standby ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="_histoFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>491</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>DACs</string>
     </attribute>
     <widget class="QFrame" name="_DACScanFrame">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>431</width>
        <height>331</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>291</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="dac17Label">
         <property name="text">
          <string>I_Delay</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="spinBox_8"/>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="dac3hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="dacV17Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="dac7hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="dac21Label">
         <property name="text">
          <string>V_Gnd</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="dacV21Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QSpinBox" name="spinBox_19"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="dac22Label">
         <property name="text">
          <string>V_Tp_ref</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSlider" name="dac11hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QSpinBox" name="spinBox_27"/>
       </item>
       <item row="11" column="3">
        <widget class="QLabel" name="dacV25Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QSpinBox" name="spinBox_26"/>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="dacV24Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="dac25Label">
         <property name="text">
          <string>V_Tp_refA</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="spinBox_24"/>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="dacV18Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="dac4hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="dac18Label">
         <property name="text">
          <string>I_TP_BufferIn</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="dac23Label">
         <property name="text">
          <string>V_Fbk</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSlider" name="dac9hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QSpinBox" name="spinBox_25"/>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="dac6hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="spinBox_11"/>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="dacV20Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSlider" name="dac8hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QSpinBox" name="spinBox_20"/>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="dacV22Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_7"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dac15Label">
         <property name="text">
          <string>I_DAC_test</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_23"/>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="dac1hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="dac26Label">
         <property name="text">
          <string>V_Tp_refB</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSlider" name="dac12hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QSpinBox" name="spinBox_12"/>
       </item>
       <item row="12" column="3">
        <widget class="QLabel" name="dacV26Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="dac19Label">
         <property name="text">
          <string>I_TP_BufferOut</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="spinBox_9"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dac16Label">
         <property name="text">
          <string>I_DAC_DiscH</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="dacV15Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_6"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="dacV14Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="dacV19Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="dac5hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="dac20Label">
         <property name="text">
          <string>V_Rpz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="dacV16Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="dac2hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="dac14Label">
         <property name="text">
          <string>I_DAC_DiscL</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="dac0hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="dac24Label">
         <property name="text">
          <string>V_Cas</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSlider" name="dac10hSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QLabel" name="dacV23Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>20</width>
        <height>411</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>416</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="dac0Label">
         <property name="text">
          <string>Threshold0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="dac0hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_5"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="dacV0Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dac1Label">
         <property name="text">
          <string>Threshold1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="dac1hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_22"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="dacV1Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dac2Label">
         <property name="text">
          <string>Threshold2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="dac2hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_3"/>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="dacV2Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="dac3Label">
         <property name="text">
          <string>Threshold3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="dac3hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="spinBox_4"/>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="dacV3Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="dac4Label">
         <property name="text">
          <string>Threshold4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="dac4hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="spinBox_21"/>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="dacV4Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="dac5Label">
         <property name="text">
          <string>Threshold5</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="dac5hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="dacV5Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="dac6Label">
         <property name="text">
          <string>Threshold6</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="dac6hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="spinBox_10"/>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="dacV6Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="dac7Label">
         <property name="text">
          <string>Threshold7</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="dac7hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QSpinBox" name="spinBox_14"/>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="dacV7Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="dac8Label">
         <property name="text">
          <string>I_Preamp</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSlider" name="dac8hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QSpinBox" name="spinBox_15"/>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="dacV8Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="dac9Label">
         <property name="text">
          <string>I_Ikrum</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSlider" name="dac9hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QSpinBox" name="spinBox_16"/>
       </item>
       <item row="9" column="3">
        <widget class="QLabel" name="dacV9Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="dac10Label">
         <property name="text">
          <string>I_Shaper</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSlider" name="dac10hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QSpinBox" name="spinBox_17"/>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="dacV10Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="dac11Label">
         <property name="text">
          <string>I_Disc</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSlider" name="dac11hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QSpinBox" name="spinBox_18"/>
       </item>
       <item row="11" column="3">
        <widget class="QLabel" name="dacV11Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="dac12Label">
         <property name="text">
          <string>I_Disc_LS</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSlider" name="dac12hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="12" column="3">
        <widget class="QLabel" name="dacV12Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="dac13Label">
         <property name="text">
          <string>I_Shaper_test</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QSlider" name="dac13hSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QSpinBox" name="spinBox_13"/>
       </item>
       <item row="13" column="3">
        <widget class="QLabel" name="dacV13Label">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMpx3Eq">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMpx3Eq"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
